Game/Game/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Game;
+using IConsoleWrapper = ConsoleWrapper.IConsoleWrapper;
+
+IConsoleWrapper console = new ConsoleWrapper.ConsoleWrapper();
```

Comment:
```suggestion
var console = new ConsoleWrapper.ConsoleWrapper();
```
Вам тут не важно, что это именно интерфейс, он при вызове Game.Game(console) сам скастается